esdk-performance-testing/results/raw-data/python_results.json

```diff
@@ -0,0 +1,1308 @@
+{
+  "metadata": {
+    "language": "python",
+    "timestamp": "2025-09-02 17:01:18",
+    "total_tests": 54,
+    "python_version": "3.12.7",
+    "cpu_count": 12,
+    "total_memory_gb": 36.0
+  },
+  "results": [
+    {
+      "test_name": "throughput",
+      "language": "python",
+      "data_size": 1024,
+      "algorithm_suite": "default",
+      "frame_length": null,
+      "concurrency": 1,
+      "encrypt_latency_ms": 1.9099712371826172,
+      "decrypt_latency_ms": 4.344916343688965,
+      "end_to_end_latency_ms": 6.2569379806518555,
+      "ops_per_second": 159.82258464000608,
+      "bytes_per_second": 163658.32667136623,
+      "peak_memory_mb": 0.0,
+      "avg_memory_mb": 0.0,
+      "cumulative_allocations_mb": 0.0,
+      "memory_efficiency_ratio": 0.0,
```

Comment:
Is there a README or similar that explains the results JSON? Some fields are obvious but things like this could use an explanation

Reply:
There's a README but added a bit more info.